### ISSUE TITLE
Extra dot at the end in Discovery/DNS/

### ISSUE THREAD

**Issue by pansa** [danielmiessler/SecLists#138]
subdomains-top1mil-*.txt contains some lines with extra dot at the end, e.g

$ grep -E "\.$" subdomains-top1mil-110000.txt 
m.
ns2.cl.bellsouth.net.
ns1.viviotech.net.
ns2.viviotech.net.
ns3.cl.bellsouth.net.
ferrari.fortwayne.com.
jordan.fortwayne.com.
quatro.oweb.com.
c.ns.emailvision.net.
a.ns.emailvision.net.
b.ns.emailvision.net.
d.ns.emailvision.net.
mail.
...

Its drive to wrong concatination result.

**Comment by g0tmi1k** [comment]
```
$ for x in *; do                                                                                                                                               echo $x
  grep -E "\.$" $x | wc -l
  echo
done

bitquark-subdomains-top100K.txt
       0

deepmagic.com-top500prefixes.txt
       0

deepmagic.com-top50kprefixes.txt
       0

fierce-hostlist.txt
       0

namelist.txt
       0

sortedcombied-knock-dnsrecon-fierce-reconng.txt
       8

subdomains-top1mil-110000.txt
     101

subdomains-top1mil-20000.txt
      24

subdomains-top1mil-5000.txt
       8

$
```

**Comment by g0tmi1k** [comment]
2x part fix in #180
